core/core-funcs.el

```diff
@@ -140,14 +140,29 @@ Supported properties:
        `((dolist (val ',def-key)
           (define-key (eval (car val)) (kbd (cdr val)) ',func)))))))
 
+(defun spacemacs/prettify-org-buffer ()
```

Comment:
@JAremko I was going to report on table alignment issue. Good that you know it already.